### ISSUE TITLE
Current accessibility issues

### ISSUE THREAD

**Issue by beqabeqa473** [UnigramDev/Unigram#2653]
Currently, several accessibility issues present in unigram that would be good to fix.

1. empty items present when navigating with tab in the unigram main window.
2. In video chat window, state of mute/unmute button is not reflected when pressing, but label next to it changes. This label should be presented for screenreaders as button label.
3. In video hat, button to open more options is not labeled. There is corresponding string in telegram translations, so it should be presented.
4. In audio player, slider for seeking have no accessibility support. It should implement role of slider as well as controlling from keyboard.
